### DISCUSSION TITLE
Using SPI1 instead of SPI0 with the SD card library

### DISCUSSION THREAD

**Discussion by NatanielSlo** [earlephilhower/arduino-pico#1035] [Q&A]
So i am trying to connect and then launch a SD card reader to raspberry pico using SPI1 and not the default SPI0 one because the default one is taken by a LoRa module.

```
  SPI1.setRX(12);
  SPI1.setTX(15);
  SPI1.setSCK(14); 
  SPI1.setCS(13);
```

I figured out this the way to define my SPI1 pins, but i still have no clue how to launch the SD card library on SPI1 and not the default SPI0.

I know that the defining code is right because i tested the sd card module on alternative SPI0 pins and it worked,

But i still have no clue on how to run a SPI device on SPI1

Please have mercy i am a beginner.

**Comment by earlephilhower** [comment]
Check out https://arduino-pico.readthedocs.io/en/latest/fs.html#using-second-spi-port-for-sd
